### PR TITLE
Made changes in unit tests to support the new output changes

### PR DESCRIPTION
**Pull request by abhijitpal1247**

### CODE REVIEW

**Review by rahm-decypher — APPROVED** [review]
looks good. @KimiJL feel free to take a quick look. thanks

File: examples/configs/autoguard/autoguard_config/flows.co [hunk 1]
```diff
@@ -1,42 +1,28 @@
...
-    bot respond pii input
+  $output_result = execute autoguard_output_api(show_autoguard_message=True)
 
   if $output_result["guardrails_triggered"]
-    $autoguard_output_response = $output_result['combined_response']
-    bot refuse to respond autoguard output
+    bot refuse to respond
     stop
   else
+    $pii_message_output = $output_result["pii_fast"]["response"]
```
Comment (KimiJL) [inline examples/configs/autoguard/autoguard_config/flows.co:17]: would this break if no guardrails triggered and pii_fast is turned off?

File: nemoguardrails/library/autoguard/README.md [hunk 2]
```diff
@@ -273,48 +273,44 @@ The colang file has to be in the following format:
...
+  $output_result = execute autoguard_output_api(show_autoguard_message=True)
+
   if $output_result["guardrails_triggered"]
-    $autoguard_output_response = $output_result['combined_response']
-    bot refuse to respond autoguard output
+    bot refuse to respond
     stop
   else
-    bot respond to question
+    $pii_message_output = $output_result["pii_fast"]["response"]
```
Comment (KimiJL) [inline nemoguardrails/library/autoguard/README.md:291]: same here, would this create a key error if pii_fast isn't enabled?

File: examples/configs/autoguard/autoguard_config/flows.co (hunk 1 above)
Comment (abhijitpal1247) [reply examples/configs/autoguard/autoguard_config/flows.co:17]: I have kept a default "pii_fast" response within result
https://github.com/decypher-ai/NeMo-Guardrails/blob/8a8f3767d83e1178961786b1a5eb109de37e0983/nemoguardrails/library/autoguard/actions.py#L87

File: nemoguardrails/library/autoguard/README.md (hunk 2 above)
Comment (abhijitpal1247) [reply nemoguardrails/library/autoguard/README.md:291]: I have kept a default pii response
https://github.com/decypher-ai/NeMo-Guardrails/blob/8a8f3767d83e1178961786b1a5eb109de37e0983/nemoguardrails/library/autoguard/actions.py#L87

**Review by KimiJL — APPROVED** [review]

**Pull request merged**